folly/portability/GFlags.h

```diff
@@ -62,9 +64,11 @@
 #define DEFINE_string(_name, _default, _description) \
   FOLLY_DEFINE_FLAG(std::string, S, _name, _default)
 
-namespace google {
+namespace gflags {
 class FlagSaver {};
-} // namespace google
+
+using int32 = int32_t;
```

Comment:
We should now be able to undo adding this alias after 581e3369a3e44ba7ed4b33114ced5ff6bf5ff7d9.

Reply:
Roger. Pull request updated.